### DISCUSSION TITLE
Is Attribute Based Access Control [Mar 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5278] [Discord Threads]
> 🙋‍♂️ **Shrey**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Shrey**  
#### @anyone ?  
#  
> 🙋‍♂️ **wkw**  
#### row-level access control is not available  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1069816371963252887)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
